pkg/standalone/stop_windows.go

```diff
@@ -29,7 +29,21 @@ func Stop(appID string) error {
 
 	for _, a := range apps {
 		if a.AppID == appID {
-			eventName, _ := syscall.UTF16FromString(fmt.Sprintf("dapr_cli_%v", a.PID))
+			var pid string
+
+			// Unlike Linux/Mac, we can't just send sigterm. In order for 'dapr shutdown'
+			// to be able to receive the signal gracefully, we must send a named event in Windows.
+			//
+			// Kill the Daprd process if Daprd was started without CLI, otherwise
+			// kill the CLI process which also kills the associated Daprd process.
+			// TODO: who handles dapr_daprd_ named events?
+			if a.CliPID == 0 {
+				pid = fmt.Sprintf("dapr_daprd_%v", a.DaprdPID)
```

Comment:
FYI, this does not work as of now. We need someone to also listen for it, the same way as `dapr_cli_{pid}`.

Reply:
The updated version, with `taskkill` seems to be working. I will get to testing it more.